conf/remove_sso_conf.py

```diff
@@ -2,7 +2,14 @@
 
 with open("/etc/ssowat/conf.json.persistent", "r", encoding='utf-8') as jsonFile:
```

Comment:
You could add these lines:

```
domain = sys.argv[1]
server_name = sys.argv[2]
```
And specify the domain to remove

Reply:
Firstly I thought that it wasn't needed, but after thinking about the multi-instance, yes it's need. Should be fixed now